ui/webui/test/README.rst

```diff
@@ -134,6 +138,12 @@ connect to the test VMs by typing `ssh test-updates`::
         Hostname 127.0.0.2
         Port 2201
         User root
+        StrictHostKeyChecking=no
+        UserKnownHostsFile=/dev/null
+
+If you are creating the file, you must also set correct permissions or SSH will not start::
+
+    chmod 600 ~/.ssh/config
```

Comment:
I never had this problem. Do you think it's something specific to your machine?

Reply:
This shouldn't be required the .ssh and content should always be 600 otherwise it's a system issue.